app/policies/current_day_policy.rb

```diff
@@ -0,0 +1,13 @@
+class CurrentDayPolicy
```

Comment:
That's a little bit overkill to separate policies by one condition.

Prefer to use structure similar to `Pundit` gem, when you have `current_user` and `resource` as instance attributes. This way you can check almost anything you want and wrap conditions in predicate methods.

First of all create `ApplicationPolicy` and separate policies by model names

[source](https://github.com/elabs/pundit)
*ALERT!* 
Dont include `Pundit` in `ApplicationController` if you are going to use `Pundit` instead your policies. That's also an authorization gem, so it can have unpredictable behaviour working with `Devise`